app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewHolderFactory.kt

```diff
@@ -327,3 +342,23 @@ class SectionLabelHeaderViewHolder(private val viewBinding: ItemTxSectionHeaderB
     }
 }
 
+class ConflictViewHolder(private val viewBinding: ItemTxConflictTxBinding, private val factory: TransactionViewHolderFactory) :
+    BaseTransactionViewHolder<TransactionView.Conflict>(viewBinding) {
+
+    private lateinit var conflictView: TransactionView.Conflict
```

Comment:
Having this internal state here is a bit risky, in case the viewHolder gets recycled on a quick scroll and the method `hasNext()` gets called before this is updated to the correct value. I think it is an edge case, but it could be a source of UI problems.

Reply:
I made a separate ticket to investigate and try to fix here: https://github.com/gnosis/safe-android/issues/1159

Reply:
Sounds good!